### ISSUE TITLE
Incompatible with newer FastAPI versions

### ISSUE THREAD

**Issue by rassie** [Kludex/flake8-fastapi#21]
It is currently impossible to add `flake8-fastapi` to a project using a newer (i.e. current, 0.7x) version of FastAPI: 
```
$ poetry add -D flake8-fastapi
Using version ^0.6.1 for flake8-fastapi

Updating dependencies
Resolving dependencies... (0.3s)

  SolverProblemError

  Because no versions of flake8-fastapi match >0.6.1,<0.7.0
   and flake8-fastapi (0.6.1) depends on fastapi (>=0.65.1,<0.66.0), flake8-fastapi (>=0.6.1,<0.7.0) requires fastapi (>=0.65.1,<0.66.0).
```

**Comment by Kludex** [comment]
I'm going to improve this package. Thanks for the feedback. :)

**Comment by Kludex** [comment]
This will be solved in `flake8-fastapi` 0.7.0. I'll release it soon.
